### PR TITLE
Translate Concept part from English to Chinese

### PR DESCRIPTION
**Pull request by cheimu**
Translate concept.md to Chinese

### CODE REVIEW

File: i18n/zh/docusaurus-plugin-content-docs/current/concepts.md [hunk 1]
```diff
@@ -1,37 +1,37 @@
 ---
-title:  How it Works
+title:  核心概念
 ---
 
-*"KubeVela is a scalable way to create PaaS-like experience on Kubernetes"*
+*"KubeVela是一个面向平台构建者的、简单易用但又高度可拓展的基于Kubernetes的云原生平台构建引擎。"*
```
Comment (wonderflow) [inline i18n/zh/docusaurus-plugin-content-docs/current/concepts.md]:
```suggestion
*"KubeVela 是一个面向平台构建者基于 Kubernetes 构建易用可拓展的云原生应用平台的引擎。"*
```

File: i18n/zh/docusaurus-plugin-content-docs/current/concepts.md [hunk 2]
```diff
@@ -1,37 +1,37 @@
...
 
-In this documentation, we will explain the core idea of KubeVela and clarify some technical terms that are widely used in the project.
+在本部分中，我们会对KubeVela的核心思想进行详细解释，并进一步阐清一些在本项目中被广泛使用的技术术语。
```
Comment (wonderflow) [inline i18n/zh/docusaurus-plugin-content-docs/current/concepts.md]:
```suggestion
在本部分中，我们会对 KubeVela 的核心思想进行详细解释，并进一步阐清一些在本项目中被广泛使用的技术术语。
```

File: i18n/zh/docusaurus-plugin-content-docs/current/concepts.md [hunk 3]
```diff
@@ -1,37 +1,37 @@
...
+- **业务用户**
+  - 选择部署环境、模型和可用模块来组装应用，并把应用部署到目标环境中。
 
-Below is how this workflow looks like:
+工作流如下图所示:
 
 ![alt](resources/how-it-works.png)
 
-This template based workflow make it possible for platform team enforce best practices and deployment confidence with a set of Kubernetes CRDs, and give end users a *PaaS-like* experience (*i.e. app-centric, higher level abstractions, self-service operations etc*) by natural.
+这种基于模板的工作流，通过使用一系列的 Kubernetes 的 CRD ，使平台团队能够执行最佳的实践和 deployment confidence 成为现实，并且可以很自然地为业务用户提供平台即服务的体验（比如：app-centric， higher level abstractions， self-service operations）。
```
Comment (wonderflow) [inline i18n/zh/docusaurus-plugin-content-docs/current/concepts.md]:
```suggestion
这种基于模板的工作流使得平台团队能够在一系列的 Kubernetes CRD 之上，引导用户遵守他们构建的最佳实践和 部署经验，并且可以很自然地为业务用户提供 PaaS 级别的体验（比如：“以应用为中心”，“高层次的抽象”，“自助式运维操作”等等）。
```

File: i18n/zh/docusaurus-plugin-content-docs/current/concepts.md [hunk 4]
```diff
@@ -1,37 +1,37 @@
...
 
 ![alt](resources/what-is-kubevela.png)
 
-Below are the core concepts in KubeVela that make this happen.
+下面开始介绍 KubeVela 的核心概念
 
 ## `Application`
-The *Application* is the core API of KubeVela. It allows developers to work with a single artifact to capture the complete application deployment with simplified primitives.
+*Application*，应用，是 KubeVela 的核心API。它允许开发者通过一个单一的 artifact 来使用简化的 primitives 来描述完整的应用部署。
```
Comment (wonderflow) [inline i18n/zh/docusaurus-plugin-content-docs/current/concepts.md]:
```suggestion
应用（*Application*），是 KubeVela 的核心 API。它使得业务开发者只需要基于一个单一的制品和一些简单的原语就可以构建完整的应用。
```

File: i18n/zh/docusaurus-plugin-content-docs/current/concepts.md [hunk 5]
```diff
@@ -1,37 +1,37 @@
...
 
-In application delivery platform, having an "application" concept is important to simplify administrative tasks and can serve as an anchor to avoid configuration drifts during operation. Also, it provides a much simpler path for on-boarding Kubernetes capabilities to application delivery process without relying on low level details. For example, a developer will be able to model a "web service" without defining a detailed Kubernetes Deployment + Service combo each time, or claim the auto-scaling requirements without referring to the underlying KEDA ScaleObject.
+在应用交付平台，为了简化运维任务，并且作为避免操作过程中产生配置漂移的固定点，*Application*这个概念尤为重要。同时，此概念为将 Kubernetes的能力引入应用交付过程提供了一个更简单的，不用依赖底层细节的途径。 举个例子，开发者能够不需要每次都定义一个详细的 Kubernetes Deployment + Service 的组合来建模一个 web service ，或者不用依靠底层的 KEDA ScaleObject 来获取自动扩容的需求。
```
Comment (wonderflow) [inline i18n/zh/docusaurus-plugin-content-docs/current/concepts.md]:
```suggestion
在应用交付平台中，有一个 *Application* 的概念尤为重要，因为这可以很大程度上简化运维任务，并且作为一个锚点避免操作过程中产生配置漂移的问题。同时，它也帮助应用交付过程中引入 Kubernetes的能力提供了一个更简单的、且不用依赖底层细节的途径。 举个例子，开发者能够不需要每次都定义一个详细的 Kubernetes Deployment + Service 的组合来建模一个 web service ，或者不用依靠底层的 KEDA ScaleObject 来获取自动扩容的需求。
```

File: i18n/zh/docusaurus-plugin-content-docs/current/concepts.md (hunk 1 above)
Comment (cheimu) [reply i18n/zh/docusaurus-plugin-content-docs/current/concepts.md]: Add spaces and exchange words' position

File: i18n/zh/docusaurus-plugin-content-docs/current/concepts.md (hunk 2 above)
Comment (cheimu) [reply i18n/zh/docusaurus-plugin-content-docs/current/concepts.md]: spaces added

File: i18n/zh/docusaurus-plugin-content-docs/current/concepts.md (hunk 3 above)
Comment (cheimu) [reply i18n/zh/docusaurus-plugin-content-docs/current/concepts.md]: Done.

File: i18n/zh/docusaurus-plugin-content-docs/current/concepts.md (hunk 5 above)
Comment (cheimu) [reply i18n/zh/docusaurus-plugin-content-docs/current/concepts.md]: Done.

File: i18n/zh/docusaurus-plugin-content-docs/current/concepts.md (hunk 4 above)
Comment (cheimu) [reply i18n/zh/docusaurus-plugin-content-docs/current/concepts.md]: Done.

**Review by wonderflow — COMMENTED** [review]
英文单词空格调整下，其他没问题 \cc @sunny0826

**Review by Ghostbaby — APPROVED** [review]
lgtm

**Review by wonderflow — APPROVED** [review]

**Pull request merged**